src/group_theory/subgroup.lean

```diff
@@ -1227,6 +1222,26 @@ le_antisymm
 
 end monoid_hom
 
+namespace subgroup
+
+variables {N : Type*} [group N] (H : subgroup G)
+
+@[to_additive] lemma map_eq_bot_iff {f : G →* N} : H.map f = ⊥ ↔ H ≤ f.ker :=
```

Comment:
I don't know what the convention is (let's let someone else decide), but this would enable `rw H.map_eq_bot_iff` which might be useful.
```suggestion
@[to_additive] lemma map_eq_bot_iff (f : G →* N) : H.map f = ⊥ ↔ H ≤ f.ker :=
```